po/es.po

```diff
@@ -9,68 +9,67 @@ msgstr ""
 "Project-Id-Version: xdg-desktop-portal-gtk\n"
 "Report-Msgid-Bugs-To: \n"
 "POT-Creation-Date: 2018-02-14 12:37+0100\n"
-"PO-Revision-Date: 2016-06-28 01:17+0100\n"
-"Last-Translator: Mario Sanchez Prada <mario@endlessm.com>\n"
+"PO-Revision-Date: 2018-02-18 01:22-0600\n"
+"Last-Translator: Adolfo Jayme Barrientos <fitojb@ubuntu.com>\n"
 "Language-Team: Spanish; Castilian <>\n"
 "Language: es\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
-"X-Generator: Gtranslator 2.91.7\n"
+"X-Generator: Poedit 2.0.6\n"
 
 #: data/xdg-desktop-portal-gtk.desktop.in.in:4
 msgid "Portal"
-msgstr ""
+msgstr "Portal"
 
 #. TRANSLATORS: Don't translate this text (this is icon name)
 #: data/xdg-desktop-portal-gtk.desktop.in.in:6
 msgid "applications-system-symbolic"
-msgstr ""
+msgstr "applications-system-symbolic"
 
 #: src/access.c:271
 msgid "Deny Access"
-msgstr ""
+msgstr "Denegar el acceso"
 
 #: src/access.c:274
 msgid "Grant Access"
-msgstr ""
+msgstr "Conceder el acceso"
```

Comment:
Ditto about using Header capitalization (and similar "soft" suggestion to consider using "Conceder Acceso")